### ISSUE TITLE
False Positive: Can not move, Does not implement Display\Debug

### ISSUE THREAD

**Issue by Liyixin95** [intellij-rust/intellij-rust#9697]
## Environment

* **IntelliJ Rust plugin version:** 0.4.182.4996-222
* **Rust toolchain version:** 1.65.0 (897e37553 2022-11-02) x86_64-pc-windows-gnu
* **IDE name and version:** CLion 2022.2.4 (CL-222.4345.21)
* **Operating system:** Windows 11 10.0
* **Macro expansion:** enabled
* **Additional experimental features:** org.rust.macros.proc

## Problem description
![image](https://user-images.githubusercontent.com/10390125/200470929-9b90c4ee-4612-44e6-b734-ad5db56f046a.png)

![image](https://user-images.githubusercontent.com/10390125/200471197-c23ede24-b03b-4b3e-8228-a9aa764c8248.png)

## Steps to reproduce

```rust
#[cfg(test)]
mod tests {
    use std::sync::{Arc, RwLock};

    #[test]
    fn test() {
        let lock = Arc::new(RwLock::new(12));
        let i = *lock.read().unwrap();
        eprintln!("i = {:#?}", i);
        eprintln!("i = {}", i);
    }
}
```

**Comment by neonaot** [comment]
Hi! 
Do you still have this (or a similar) problem? 
I cannot reproduce it on the same versions on Windows too, it seems like the plugin doesn't see needed traits/implementations from stdlib.

**Comment by Liyixin95** [comment]
> Hi! Do you still have this (or a similar) problem? I cannot reproduce it on the same versions on Windows too, it seems like the plugin doesn't see needed traits/implementations from stdlib.

This problem seems to be fixed on 0.4.185.5086-223, and Clion Version: 2022.3(CL-223.7571.171). Thanks
